src/controllers/auth.js

```diff
@@ -0,0 +1,9 @@
+exports.callback = async (req, res) => {
+  res.json(req.user);
```

Comment:
You can also set the user into the session here as: `req.session.user = req.user`

Reply:
passport.serializeUser and passport.deserializeUser middlewares in the google strategy file puts users to session.passport.user . Do I still add it to session.user?

Reply:
Ah okay makes sense, to let passport js handle the session data.